index.js

```diff
@@ -1,46 +1,80 @@
 'use strict'
 
+const _ = require('lodash')
+const fs = require('fs-extra')
 const readPkg = require('read-pkg-up')
 const truncate = require('cli-truncate')
 const wrap = require('wrap-ansi')
 const pad = require('pad')
+const fuse = require('fuse.js')
+const Promise = require('bluebird')
+const homeDir = require('home-dir')
 const types = require('./lib/types')
 
-function getEmojiChoices(types) {
+function getEmojiChoices(types, symbol) {
```

Comment:
Let's get the config directly and destructure it:
`function getEmojiChoices({ types, symbol }) {`